README.md

```diff
@@ -315,16 +266,260 @@ For example:
   - `fields, fields->>'c2', sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields, sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields->>'c2', sqrt((fields->>'c1')::int)`: there is no fields jsonb column, so function `sqrt()` is pushed down.
+  
+Also see [Limitations](#limitations).  
+
+Supported platforms
+-------------------
 
-## Contributing
+`influxdb_fdw` was developed on Linux, and should run on any
+reasonably POSIX-compliant system.
+
+`influxdb_fdw` is designed to be compatible with PostgreSQL 10 ~ 14.
+
+Go version should be 1.10.4 or later.
+
+Installation
+------------
+### Prerequisites
+
+Install InfluxDB Go client library
+<pre>
+go get github.com/influxdata/influxdb1-client/v2
+</pre>
+
+### Source installation
+
+Add a directory of pg_config to PATH and build and install influxdb_fdw.
+<pre>
+make USE_PGXS=1 with_llvm=no
+make install USE_PGXS=1 with_llvm=no
+</pre>
+with_llvm=no is necessary to disable llvm bit code generation when PostgreSQL is configured with --with-llvm because influxdb_fdw use go code and cannot be compiled to llvm bit code.
+
+If you want to build influxdb_fdw in a source tree of PostgreSQL instead, use
+<pre>
+make with_llvm=no
+make install  with_llvm=no
+</pre>
+
+Usage
+-----
+
+## CREATE SERVER options
+
+`influxdb_fdw` accepts the following options via the `CREATE SERVER` command:
+
+- **dbname** as *string*, optional
+
+- **host** as *string*, optional
+
+- **port** as *integer*, optional
+
+
+## CREATE USER MAPPING options
+
+`influxdb_fdw` accepts the following options via the `CREATE USER MAPPING`
```

Comment:
Could you please update the new options as well?
there are two authorization options when connect to InfluxDB version 1.x and 2.x
'user', 'password' for version 1.x
'auth_token' for version 2.x

----

#### Go Client connect to InfluxDB ver 1.x
<pre>
CREATE USER MAPPING FOR CURRENT_USER SERVER influxdb_server OPTIONS(user 'user', password 'pass');
</pre>
#### Influxdb_cxx Client connect to InfluxDB ver 1.x
<pre>
CREATE USER MAPPING FOR CURRENT_USER SERVER influxdb_server OPTIONS(user 'user', password 'pass');
</pre>
#### Influxdb_cxx Client connect to InfluxDB ver 2.x
<pre>
CREATE USER MAPPING FOR CURRENT_USER SERVER influxdb_server OPTIONS(auth_token 'token');
</pre>

Reply:
Added in EXAMPLES section and also in USAGE section.

Reply:
Thank you, both section checked.